sjb/config/common/test_cases/origin_installed_release.yml

```diff
@@ -29,21 +29,6 @@ extensions:
            sudo mv "${plugin}.py" /usr/share/ansible/plugins/callback
         done
         sudo sed -r -i -e 's/^#?stdout_callback.*/stdout_callback = default_with_output_lists/' -e 's/^#?callback_whitelist.*/callback_whitelist = generate_junit/' /etc/ansible/ansible.cfg
-    - type: "script"
-      title: "determine the release commit for origin images and version for rpms"
-      timeout: 600
-      repository: "origin"
-      script: |-
-        jobs_repo="/data/src/github.com/openshift/aos-cd-jobs/"
-        git log -1 --pretty=%h > "${jobs_repo}/ORIGIN_COMMIT"
-
-        source hack/lib/init.sh
-        os::build::rpm::get_nvra_vars
-        echo "-${OS_RPM_VERSION}-${OS_RPM_RELEASE}" > "${jobs_repo}/ORIGIN_PKG_VERSION"
-        echo "${OS_GIT_MAJOR}.${OS_GIT_MINOR}" | sed "s/+//" > "${jobs_repo}/ORIGIN_RELEASE"
-        echo "${OS_RPM_VERSION}" | cut -d'.' -f2 > "${jobs_repo}/ORIGIN_PKG_MINOR_VERSION"
-        tag="$( echo "v${OS_GIT_MAJOR}.${OS_GIT_MINOR}" | sed "s/+//" )"
-        echo "${tag}" > "${jobs_repo}/ORIGIN_TAG"
```

Comment:
removed - duplicates script in [common/test_cases/origin_release.yml](https://github.com/openshift/aos-cd-jobs/blob/master/sjb/config/common/test_cases/origin_release.yml)

Reply:
cool